firmware/user/utils/spi_mem/nvm_interface.h

```diff
@@ -25,6 +25,20 @@ typedef struct
     int32_t actionsTaken;
 } demonMemorial_t;
 
+
+#define NUM_FLIGHTSIM_TOP_SCORES 4
+#define FLIGHT_HIGH_SCORE_NAME_LEN 4
+
+typedef struct
+{
+    //One set for any% one set for 100%
+    char displayName[NUM_FLIGHTSIM_TOP_SCORES*2][FLIGHT_HIGH_SCORE_NAME_LEN];
+    uint32_t timeCentiseconds[NUM_FLIGHTSIM_TOP_SCORES*2];
+    uint8_t flightInvertY;
+    uint8_t reserved;
+    uint16_t reserved2; //pad to 32 bit.
```

Comment:
Padding to 32 bit is nice, but decorating the struct `__attribute__((aligned(4)))` is a little nicer. `settings_t` is already decorated that way.

Reply:
Note to self, I should align all the sub-structs there